packages/sdk/src/services/editSwap/editSwap.ts

```diff
@@ -0,0 +1,189 @@
+import { BigNumber, ContractReceipt } from 'ethers';
+import {
+  estimateGas,
+  executeTransaction,
+  simulateTx,
+} from '../executeTransaction';
+import { encodeSwap } from '../swap/encode';
+import {
+  CompleteEditSwapDetails,
+  EditSwapArgs,
+  EditSwapPeripheryParameters,
+} from './types';
+import { getEditSwapPeripheryParams } from './getEditSwapPeripheryParams';
+import { getTokenDetails, scale } from '@voltz-protocol/commons-v2';
+import {
+  baseAmountToNotionalBN,
+  notionalToBaseAmount,
+} from '../../utils/helpers';
+import { VERY_BIG_NUMBER, ZERO_BN } from '../../utils/constants';
+import { fixedRateToPrice } from '../../utils/math/tickHelpers';
+import { decodeSwapOutput, InfoPostSwap, processInfoPostSwap } from '../swap';
+
+async function createEditSwapParams({
+  poolId,
```

Comment:
can we avoid the need for poolId in context of edit swap and derive it from the position (see v1-sdk editSwap)